src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/ElectricUtilitySpell.java

```diff
@@ -43,14 +43,14 @@
 @Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
 public class ElectricUtilitySpell extends PassiveSpellItem {
 
-    private final int totalManaReduction;
+    private final int manaReduction;
```

Comment:
If we're editing all spells anyway, can you change any individual values like this to a final IConfig value? Getting the spell members from an IConfig allows players to edit the values during playtime; final member variables require a reload of the world

Reply:
Done.